libsam.c

```diff
@@ -511,7 +510,7 @@ int sam_put_grp_members_sid(struct hive *hdesc, int grp, struct sid_array *sarra
 
      if (gverbose) printf("put_grp_members_sid: ajusted: mofs = %x, mlen = %x (%d)\n", mofs + 0x34 ,mlen,mlen);
 
-     if (gverbose) hexdump(&c->data, 0, c->len, 1);
+     if (gverbose) hexdump((char *)&(c->data), 0, c->len, 1);
```

Comment:
Can you explain why you change `&` into `(char *)&(` ?
Either as an in-code comment (better) or in the commit description.